### DISCUSSION TITLE
[README] 튜닝의 끝은 순정! 레트로 리드미 이력서

### DISCUSSION THREAD

**Discussion by saejinpark** [DeveloperAcademy-POSTECH/.github#24] [Idea]
<table>
  <caption>
    이력서 양식
  </caption>
  <thead>
    <tr></tr>
    <tr>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
    </tr>
    <tr>  
        <td rowspan="8" colspan="2" align="center">
          [ 사 진 ]
        </td>
        <th colspan="10">
            <h1>
                이 력 서
            </h1>
        </th>
    </tr>
    <tr></tr>
    <tr>  
        <th colspan="3">한 글</th>
        <td colspan="3"></td>
        <th colspan="4">연 락 쳐</th>
    </tr>
    <tr></tr>
    <tr>  
        <th colspan="3">영 문</th>
        <td colspan="3"></td>
        <td colspan="4"></td>
    </tr>
    <tr></tr>
    <tr>  
        <th colspan="3">이 메 일</th>
        <td colspan="7"></td>
    </tr>
    <tr></tr>
    <tr>  
        <th colspan="2">현 주 소</th>
        <td colspan="10"></td>
    </tr>
  </thead>
  <tbody>
    <tr></tr>
    <tr>
      <th colspan="12">학 적 사 항</th>
    </tr>
    <tr>
      <th colspan="3">재 학 기 간</th>
      <th colspan="4">학 교 명</th>
      <th colspan="3">전 공</th>
      <th colspan="2">비 고</th>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr>
      <th colspan="12">실무 경력사항</th>
    </tr>
    <tr>
      <th colspan="3">근 무 기 간</th>
      <th colspan="4">근 무 지</th>
      <th colspan="3">직 위</th>
      <th colspan="2">퇴 직 사 유</th>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr>
      <th colspan="12">기타 경력사항</th>
    </tr>
    <tr>
      <th colspan="3">근 무 기 간</th>
      <th colspan="4">근 무 지</th>
      <th colspan="3">직 위</th>
      <th colspan="2">퇴 직 사 유</th>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
  </tbody>
  <tfoot>
    <tr>
      <th colspan="2">
        참 고 사 항
      </th>
      <td colspan="10"></td>
    </tr>
    <tr></tr>
    <tr>
      <td colspan="12">
        <ul>
          <p></p>
          <b>위 기재 사항은 사실과 다름이 없습니다.</b>
          <p></p>
          <h3 align="center">
            2024 년 6 월 2 일
          </h3>
          <p></p>
          <h3 align="right">
            <u>지원자 :</u>
            <span></span>
            <span>X X X</span>
            <span>(인)</span>
          </h3>
        </ul>
      </td>
    </tr>
  </tfoot>
</table>

<details>
  <summary>코드</summary>

```
  <table>
  <caption>
    이력서 양식
  </caption>
  <thead>
    <tr></tr>
    <tr>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
      <td width="120"></td>
    </tr>
    <tr>  
        <td rowspan="8" colspan="2" align="center">
          [ 사 진 ]
        </td>
        <th colspan="10">
            <h1>
                이 력 서
            </h1>
        </th>
    </tr>
    <tr></tr>
    <tr>  
        <th colspan="3">한 글</th>
        <td colspan="3"></td>
        <th colspan="4">연 락 쳐</th>
    </tr>
    <tr></tr>
    <tr>  
        <th colspan="3">영 문</th>
        <td colspan="3"></td>
        <td colspan="4"></td>
    </tr>
    <tr></tr>
    <tr>  
        <th colspan="3">이 메 일</th>
        <td colspan="7"></td>
    </tr>
    <tr></tr>
    <tr>  
        <th colspan="2">현 주 소</th>
        <td colspan="10"></td>
    </tr>
  </thead>
  <tbody>
    <tr></tr>
    <tr>
      <th colspan="12">학 적 사 항</th>
    </tr>
    <tr>
      <th colspan="3">재 학 기 간</th>
      <th colspan="4">학 교 명</th>
      <th colspan="3">전 공</th>
      <th colspan="2">비 고</th>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr>
      <th colspan="12">실무 경력사항</th>
    </tr>
    <tr>
      <th colspan="3">근 무 기 간</th>
      <th colspan="4">근 무 지</th>
      <th colspan="3">직 위</th>
      <th colspan="2">퇴 직 사 유</th>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr>
      <th colspan="12">기타 경력사항</th>
    </tr>
    <tr>
      <th colspan="3">근 무 기 간</th>
      <th colspan="4">근 무 지</th>
      <th colspan="3">직 위</th>
      <th colspan="2">퇴 직 사 유</th>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
    <tr></tr>
    <tr>
      <td align="center" colspan="3">~</td>
      <td colspan="4"></td>
      <td colspan="3"></td>
      <td colspan="2"></td>
    </tr>
  </tbody>
  <tfoot>
    <tr>
      <th colspan="2">
        참 고 사 항
      </th>
      <td colspan="10"></td>
    </tr>
    <tr></tr>
    <tr>
      <td colspan="12">
        <ul>
          <p></p>
          <b>위 기재 사항은 사실과 다름이 없습니다.</b>
          <p></p>
          <h3 align="center">
            2024 년 6 월 2 일
          </h3>
          <p></p>
          <h3 align="right">
            <u>지원자 :</u>
            <span></span>
            <span>X X X</span>
            <span>(인)</span>
          </h3>
        </ul>
      </td>
    </tr>
  </tfoot>
</table>
```

</details>

**Comment by yeniful** [comment]
우와 !! 리드미 이력서라니 XD !!

**Reply by saejinpark** [reply]
하핫!
